yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,822 @@
+        }
+      }
+    }
+    list md5 {
+      configd:help "MD5 authentication type.";
+      description "MD5 authentication type.";
+      key "password";
+      leaf password {
+        type string {
+          length "1..254";
+        }
+        mandatory true;
+        configd:help "Actual password.";
+        description "Actual password.";
+      }
+      container authenticate {
+        configd:help "Authentication .";
+        description "Authentication.";
+        leaf snp {
+          type authenticate-snp-type;
+          configd:help "SNP PDUs.";
+          description "SNP PDUs.";
+        }
+      }
+    }
+  }
+
+  grouping isis-area-domain-password {
+    container area-password {
+      description "area password.";
+      configd:help "area password.";
+      uses isis-password ;
+    }
+    container domain-password {
+      description "domain password.";
+      configd:help "domain password.";
+      uses isis-password ;
+    }
+  }
+
+  grouping redistribute-attributes {
+    configd:help "Common optional attributes of any redistribute entry.";
+    description  "Common optional attributes of any redistribute entry.";
+    choice attribute {
+      list route-map {
+        configd:help "Route map reference.";
+        description "Route map reference.";
+        key "rm-name";
+        leaf rm-name {
+          type string {
+            length "1..max";
+          }
+          configd:help "Applies the conditions of the specified route-map to routes that
+          are redistributed into this routing instance.";
+          description "Applies the conditions of the specified route-map to routes that
+          are redistributed into this routing instance.";
+        }
+        leaf metric {
+          configd:help "Metric for default route.";
+          description "Metric for default route.";
+          type uint32 {
+            range "0..16777215";
+          }
+          default "0";
+        }
+      }
+      list metric {
+        configd:help "Metric for default route.";
+        description "Metric for default route.";
+        key "metric-id";
+        leaf metric-id {
+          type uint32 {
+            range "0..16777215";
+          }
+          default "0";
+          configd:help "Metric used for the redistributed route. If 0,
+          the default-metric attribute is used instead.";
+          description "Metric used for the redistributed route. If 0,
+          the default-metric attribute is used instead.";
+        }
+        leaf route-map {
+          type string {
+            length "1..max";
+          }
+          configd:help "Route map reference.";
+          description "Route map reference";
+        }
+      }			
+    }
+  }
+
+  grouping redistribute-default {
+    container always {
+      configd:help "Always advertise default route.";
+      description "Always advertise default route.";
+      uses redistribute-attributes;
+    }
+    uses redistribute-attributes;
+  }
```

Comment:
It looks like these groupings aren't currently used? I'd suggest only adding the groupings which are currently in use/supported.

Reply:
This is Resolved.